jgiven-maven-plugin/build.gradle

```diff
@@ -58,7 +58,7 @@ generateMavenPlugin.configure {
             .collect(Collectors.toList())
     )
     onlyIf {
-        println("taskGraph: " + gradle.taskGraph.getAllTasks().stream().map(Task::getName).collect(Collectors.toList()))
+        logger.debug("taskGraph: " + gradle.taskGraph.getAllTasks().stream().map(Task::getName).collect(Collectors.toList()))
```

Comment:
The logger is given globally?

Reply:
in a sense. The logger is attached to the `project` object and thanks to grooys syntax you can shorten `project.getLogger()` to logger in this case.